third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -1950,6 +1954,140 @@ func testAccCheckComputeInstanceDestroyProducer(t *testing.T) func(s *terraform.
 	}
 }
 
+func TestComputeInstance_networkIPCustomizedDiff(t *testing.T) {
+	t.Parallel()
+
+	d := &ResourceDiffMock{
+		Before: map[string]interface{}{
+			"network_interface.#": 0,
+		},
+		After: map[string]interface{}{
+			"network_interface.#": 1,
+		},
+	}
+
+	err := forceNewIfNetworkIPNotUpdatableFunc(d);
+
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	if d.IsForceNew {
+		t.Errorf("Expected not force new if network_interface array size static")
```

Comment:
Isn't the array size not static in this example? should that be "has changed"?

Reply:
yep sorry!